<?xml version="1.0"?>
<launch>
    <node pkg="heterogeneous_formation_controller" type="hmfpc_test_node" name="hmfpc_test_node" output="screen" />
  <!-- <arg name="map_file" default="$(find heterogeneous_formation_controller)/config/map/map.yaml"/> -->
  <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)"> -->
      <!-- <param name="frame_id" value="map" /> -->
  <!-- </node> -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find heterogeneous_formation_controller)/config/test.rviz" />
</launch>